README.md

```diff
@@ -166,7 +166,8 @@ public class SomethingApiConsumerTests : IClassFixture<ConsumerMyApiPact>
   }
 
   [Fact]
-  public void GetSomething_WhenTheTesterSomethingExists_ReturnsTheSomething()
+  public void GetSometh

```

Comment:
Probably want to keep this on 1 line

Reply:
Sorry, it was a mistyped, reverted in the next commit.